### DISCUSSION TITLE
thanhle.blog/workshop-ve-on-chain-analysis

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#93] [Ideas]
# thanhle.blog/workshop-ve-on-chain-analysis

https://thanhle.blog/blog/workshop-ve-on-chain-analysis

**Comment by khiem20tc** [comment]
Workshop diễn ra khi nào vậy anh?

**Reply by thanhlmm** [reply]
a chưa biết, khi nào có lịch cụ thể a sẽ gửi qua email nha
